recaf-core/src/main/java/me/coley/recaf/Recaf.java

```diff
@@ -13,8 +13,9 @@
  *
  * @author Matt Coley
  */
-public class Recaf {
-	public static final String VERSION = "3.0.0";
+public final class Recaf {
+
+	public static final String VERSION = Recaf.class.getPackage().getSpecificationVersion();
```

Comment:
I'm just curious, how does this work? And is it viable when used in different situations? _(Running from launcher vs running from IDE)_

Reply:
> 
> 
> I'm just curious, how does this work? And is it viable when used in different situations? _(Running from launcher vs running from IDE)_

Everything is OK unless IDEA is not set up to use Gradle to run tasks.

Reply:
![setting](https://user-images.githubusercontent.com/19853368/103216806-ff420100-4927-11eb-9d8c-6e7e038b655d.png)